semgrep_output_v1.atd

```diff
@@ -787,6 +787,13 @@ type api_scans_findings <ocaml attr="deriving show"> = {
    cai_ids: string list;
 }
 
+type finding_hashes <ocaml attr="deriving show"> = {
+  start_line: string;
+  end_line: string;
+  code: string;
```

Comment:
need some comment about what code and pattern are.

Reply:
done!